style.css

```diff
@@ -0,0 +1,988 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  max-width: 1500px;
+}
```

Comment:
You need to remove this max-width. The reason is if a user who has a computer with a screen size larger than 1500px will find the webpage constrained at one side and hence having an empty space. Check the images below :arrow_down: 

You should remove that constrain and make your webpage look good without it. Remember we need to make the webpage look good in all screen sizes this is including bigger screens 

### ORIGINAL
![max width original](https://user-images.githubusercontent.com/32361963/88265642-8bdff780-ccd6-11ea-9579-8083eb7a3caf.jpg)

### YOURS
![max width](https://user-images.githubusercontent.com/32361963/88266045-33f5c080-ccd7-11ea-939b-59860bef1acb.jpg)